src/Parser/Mage/Frost/Modules/Features/FreezingRain.js

```diff
@@ -44,22 +44,22 @@ class ArcticGale extends Analyzer {
   statistic() {
     return (
       <StatisticBox
-        icon={<SpellIcon id={SPELLS.ARCTIC_GALE_TALENT.id} />}
+        icon={<SpellIcon id={SPELLS.FREEZING_RAIN_TALENT.id} />}
         value={`${formatPercentage(this.damagePercent)} %`}
-        label="Arctic Gale damage"
-        tooltip="This is the portion of your total damage attributable to Arctic Gale. This number only considers the damage bonus to Blizzard, and does not factor in extra targets you may have hit due to the increased area of effect."
+        label="Freezing Rain damage"
```

Comment:
I don't think this would give a reasonable estimate of the damage contribution in BFA since it does not account for the reduced cast time on AOE, and for single target casting it would replace another cast. I would suggest displaying something else, maybe proc chance & count?

Reply:
what proc are you talking about?

Freezing Rain is a buff that is up while Frozen Orb is up, there are no procs.

Reply:
what proc are you talking about?

Freezing Rain is a buff that is up while Frozen Orb is up, there are no procs.

Reply:
whoops for some reason I thought it was only a chance on frozen orb cast. Not sure what to display then, but DPS definitely aint accurate for the reasons mentioned above.

Reply:
Hmm, the instant cast thing while frozen orb is down is there in legion too, so its not really any different from legion aside from only giving the damage boost while frozen orb is down.

I might just remove the module entirely, ill take another look at it later.